### PR TITLE
👀npx qawolf edit & watch mode

### LINKED ISSUE #526: ✏️npx qawolf edit filename
- Add `npx qawolf edit filename` that runs a single file.
- Unless `config.watch: false`, `npx qawolf create` and `npx qawolf edit` should re-run when the file is saved.
- On re-run: close the previous browser and repl

Based on feedback from @papoola

---

Update docs as part of this.

### PR DESCRIPTION
**Pull request by jperl**
Resolves #526

### CODE REVIEW

**Comment by jperl** [comment]
After wrestling with building our own runner which works, except on windows, it is apparent we should build a jest watch plugin instead.

**Comment by jperl** [comment]
Note to self

```
DEBUG=qawolf:Run* CI=true QAW_HEADLESS=false npx jest --config=ts-jest.config.json --watch --rootDir=test/.qawolf google.test.js --detectOpenHandles --testTimeout=3600000
```

File: postInstallMessage.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const { bold } = require('kleur');
```
Comment (jperl) [inline postInstallMessage.js:1]: I moved this file to js and to the top level, so we did not need to build it before using it. This prevents an issue when trying to link locally.

File: src/cli/editCommand.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import program, { Command } from 'commander';
+import { loadConfig } from '../config';
+import { findTestPath } from '../run/findTestPath';
+import { runTests } from '../run/runTests';
+import { buildEditOptions } from '../run/buildEditOptions';
```
Comment (flaurida) [inline src/cli/editCommand.ts:5]: abc

File: src/qawolf.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  TemplateFunction,
+} from './build-code/buildTemplate';
+
+export { Config } from './config';
+
+export { create } from './create-code/create';
+
+export {
+  launch,
+  repl,
```
Comment (flaurida) [inline src/qawolf.ts:15]: abc

File: src/utils/Registry.ts [hunk 4]
```diff
@@ -0,0 +1,51 @@
+import { EventEmitter } from 'events';
+import { Browser, BrowserContext } from 'playwright';
+import { watchBrowser } from '../watch/watchBrowser';
```
Comment (flaurida) [inline src/utils/Registry.ts:3]: abc

**Pull request merged**